config/locales/en.yml

```
@@ -900,7 +900,7 @@ en:
           billion: B
           million: M
           quadrillion: Q
-          thousand: K
+          thoudand: K
```

```suggestion
          thousand: K
```

Is there any reason to change it